es/03_5_Entendiendo_El_Descriptor.md

```diff
@@ -0,0 +1,132 @@
+# 3.5: Comprensión del descriptor
+
+> :information_source: **NOTA:** Esta sección se ha agregado recientemente al curso y es un borrador inicial que aún puede estar pendiente de revisión. Lectura con precaución.
+
+Es posible que haya notado un extraño campo `desc:` en el comando `listunspent` de la sección anterior. Esto es de qué se trata (y cómo se puede usar para transferir direcciones).
+
+> :advertencia: **ADVERTENCIA DE VERSIÓN:** Esta es una innovación de Bitcoin Core v 0.17.0 que ha continuado expandiéndose a través de Bitcoin Core 0.20.0. La mayoría de los comandos de esta sección son de 0.17.0, pero el `importmulti` actualizado que admite descriptores es de 0.18.0.
+
+## Conozca la transferencia de direcciones
+
+La mayor parte de este curso supone que está trabajando completamente desde un solo nodo donde administra su propia billetera, enviando y recibiendo pagos con las direcciones creadas por esa billetera. Sin embargo, no es necesariamente así como funciona el ecosistema de Bitcoin más grande. Allí, es más probable que mueva direcciones entre billeteras e incluso configure billeteras para vigilar los fondos controlados por diferentes billeteras.
+
+Ahí es donde entran los descriptores. Son más útiles si está interactuando con _otro_ software distinto a Bitcoin Core, y realmente necesita apoyarse en este tipo de función de compatibilidad: consulte [§6.1](https://github.com/BlockchainCommons /Learning-Bitcoin-from-the-Command-Line/blob/master/06_1_Sending_a_Transaction_to_a_Multisig.md) para ver un ejemplo del mundo real de cómo tener la capacidad de los descriptores es fundamental.
```

Comment:
`[§6.1](06_1_Enviando_una_Transaccion_a_una_Direccion_Multifirma.md)`

Reply:
- `[§6.1](06_1_Enviando_una_Transaccion_a_una_Direccion_Multifirma.md)`
- I think we should prefer local paths rather than external web links.